src/main/java/Bob.java

```diff
@@ -0,0 +1,260 @@
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Scanner;
+
+public class Bob {
+    public static final String LINE_STRING = "____________________________________________________________\n";
+    public static final String DATA_DELIMITER = " @_@ ";
+    public static final String DATA_DIRECTORY = "data";
+    public static final String SAVE_FILE_NAME = "duke.txt";
+    public static final String DIRECTORY_HOME = System.getProperty("user.home");
+    public static TaskList taskList = new TaskList();
+
+
+    public static void main(String[] args) {
+        welcomeMessage();
+        loadFile();
+        scanInput();
+        goodbyeMessage();
+    }
+
+    private static void welcomeMessage() {
+        String welcome = LINE_STRING +
+                " Hello! I'm Bob 😀\n" +
+                " If you need anything hit me up fam 😌\n" +
+                LINE_STRING;
+        System.out.print(welcome);
+    }
+
+    private static void loadFile() {
+        createDataDirectory();
+        openFile();
+    }
+
+    private static void createDataDirectory() {
+        Path path = Paths.get(DIRECTORY_HOME, "data");
+        try {
+            Files.createDirectory(path);
+        } catch (IOException e) {
+            // printIOException(); TODO: figure out why IO Exception is triggered here
+        }
+    }
+
+    private static void openFile() {
+        Path path = Paths.get(DIRECTORY_HOME, DATA_DIRECTORY, SAVE_FILE_NAME);
+        try {
+            Files.createFile(path);
+        } catch (FileAlreadyExistsException  e) {
+            loadDataFile(path);
+        } catch (IOException e) {
+            printIOException();
+        }
+    }
+
+    private static void loadDataFile(Path path) {
+        try {
+            List<String> taskStrings = Files.readAllLines(path);
+            for (String taskString: taskStrings) {
+                loadTask(taskString);
+            }
+        } catch (IOException e) {
+            printIOException();
+        } catch (NoSuchMethodException e) {
+            printNoSuchMethod();
+        } catch (NumberFormatException e) {
+            printNumberFormatException();
+        }
+    }
+
+    private static void loadTask(String taskString) throws NoSuchMethodException, NumberFormatException{
+        String[] taskStringArray = taskString.split(DATA_DELIMITER);
+        Command command = getCommandType(taskStringArray[0]);
+        String isDone = taskStringArray[1];
+        String label = taskStringArray[2];
+        String time = taskStringArray[3];
+
+        switch (command) {
+        case TODO:
+            taskList.addToList(new Todo(label),false);
+            break;
+        case DEADLINE:
+            taskList.addToList(new Deadline(label, time),false);
+            break;
+        case EVENT:
+            taskList.addToList(new Event(label, time),false);
+            break;
+        }
+
+        if (Integer.parseInt(isDone) == 1) {
```

Comment:
Based on the usage here, it seems that `isDone` could deserve a better data type then `String`.